rusoto/core/src/request.rs

```diff
@@ -239,10 +240,21 @@ impl HttpClient {
         Ok(Self::from_connector_with_config(connector, config))
     }
 
+    /// An alias for [`local_agent_prepend`] for backwards compatibility
+    pub fn local_agent(&mut self, local_agent: String) {
+        self.local_agent_prepend(local_agent)
+    }
+
     /// Sets a local agent that is prepended to the default HTTP
     /// `User-Agent` used by Rusoto.
-    pub fn local_agent(&mut self, local_agent: String) {
-        self.local_agent = Some(local_agent)
+    pub fn local_agent_prepend(&mut self, local_agent: String) {
+        self.local_agent_prepend = Some(local_agent)
+    }
+
+    /// Sets a local agent that is appended to the default HTTP
+    /// `User-Agent` used by Rusoto.
+    pub fn local_agent_apppend(&mut self, local_agent: String) {
```

Comment:
@benesch caught that this should probably be...

```suggestion
    pub fn local_agent_append(&mut self, local_agent: String) {
```

Going ahead and committing that so I can merge this.

Reply:
Thank you